core/templates/dev/head/pages/exploration_editor/statistics_tab/StatisticsTab.js

```diff
@@ -245,6 +244,10 @@ oppia.controller('StatisticsTab', [
               };
             }
           ]
+        }).result.then(function(result) {
```

Comment:
Instead of adding an empty function will it be fine to use `.closed`? (Not sure whether this will handle this `modal dismiss`!)

Reply:
According to documentation:
`result` _(Type: `promise`)_ - Is resolved when a modal is closed and rejected when a modal is dismissed.
`closed` _(Type: `promise`)_ - Is resolved when a modal is closed and the animation completes.

So those 2 are not equivalent...
Instead of `then` I could use `catch` on promise but it's forbidden by current ESLint configuration .